android-todo-list/architecture-samples-main/app/src/main/java/com/statsig/todoapp/data/DefaultTaskRepository.kt

```diff
@@ -83,10 +85,42 @@ class DefaultTaskRepository @Inject constructor(
         }
     }
 
-    override fun getTasksStream(): Flow<List<Task>> {
-        return localDataSource.observeAll().map { tasks ->
-            withContext(dispatcher) {
-                tasks.toExternal()
+    override fun getTasksStream(sortOrderValue: Int): Flow<List<Task>> {
+        Statsig.logEvent(
+            "${StatsigUtil.ITEM_SORT} ",
```

Comment:
You don't need the string templating for the event name here. This is going to result in the event name not matching the others. "item_sorting " vs "item_sorting"

Reply:
I will remove it.